### PR TITLE
Buffs the fuck out of Changeling Armor

### PR DESCRIPTION
**Pull request by Iamgoofball**
Feedback says it's complete garbage for 2 DNA, so here's some proposed buffs.

Cut the cost in half

Increased the armor values of most the armor types for ling

This probably won't get accepted but at least this PR will get attention drawn to the problem.

### CODE REVIEW

**Comment by xxalpha** [comment]
Maybe you should have kept the cost at 2.

**Comment by Iamgoofball** [comment]
Let's wait for a few more comments on this before I change the cost again

File: code/game/gamemodes/changeling/powers/mutations.dm [hunk 1]
```diff
@@ -329,7 +329,7 @@
 	desc = "A tough, hard covering of black chitin with transparent chitin in front."
 	icon_state = "lingarmorhelmet"
 	flags = BLOCKHAIR | NODROP
-	armor = list(melee = 40, bullet = 15, laser = 30,energy = 10, bomb = 5, bio = 2, rad = 0)
+	armor = list(melee = 60, bullet = 30, laser = 60, energy = 30, bomb = 10, bio = 4, rad = 0)
```
Comment (GunHog) [inline code/game/gamemodes/changeling/powers/mutations.dm]: Should probably keep the values consistent across both items.

**Comment by GunHog** [comment]
The cost is fine at the new values, in my opinion.

File: code/game/gamemodes/changeling/powers/mutations.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/gamemodes/changeling/powers/mutations.dm]: Helmets always have less damage resist than the main body armor. See: security helmets vs security vests

Comment (Fox-McCloud) [inline code/game/gamemodes/changeling/powers/mutations.dm]: Uh, not exactly correct:

https://github.com/tgstation/-tg-station/blob/master/code/modules/clothing/suits/armor.dm#L18
https://github.com/tgstation/-tg-station/blob/master/code/modules/clothing/head/helmet.dm#L7

**Comment by WJohn** [comment]
I don't think the armor is the main problem people have with it, it's the slowdown that brings you down to riot-speed that's the real killer, since it's easier to get hit than at full speed. Hell the adrenaline overdose that lets you run at max game speed is basically better armor since you can yakkety sax around bullets so much more easily.

**Comment by GunHog** [comment]
The armor does not seem to have slowdown in this code, WJ.

**Comment by Iamgoofball** [comment]
@WJohn According to the Feedback thread the problem was not the slowdown but the fact that it was 2 DNA points for shitty ass armor that made you obvious a ling.

File: code/game/gamemodes/changeling/powers/mutations.dm [hunk 2]
```diff
@@ -294,7 +294,7 @@
 	desc = "We turn our skin into tough chitin to protect us from damage."
 	helptext = "Upkeep of the armor requires a low expenditure of chemicals. The armor is strong against brute force, but does not provide much protection from lasers. Retreating the armor damages our genomes. Cannot be used in lesser form."
```
Comment (GunHog) [inline code/game/gamemodes/changeling/powers/mutations.dm]: Do not forget to change the description to reflect that it is now useful against lasers.

**Comment by Fox-McCloud** [comment]
Also, this is going to cut the stun duration from tasers in half with 50 energy based armor.

**Comment by Iamgoofball** [comment]
@Fox-McCloud yep

File: code/game/gamemodes/changeling/powers/mutations.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/game/gamemodes/changeling/powers/mutations.dm]: sure

**Comment by GunHog** [comment]
Which is not all that bad a thing. It does not matter that much, as lings have anti-stuns.

**Comment by Lumipharon** [comment]
>all armour gets nerfed because non t-shirt armour is supposedly bad

>ling armour gets buffed
>it's shit because it's currently too weak!

I do agree that it's shit currently, but I find the whole situation amusing.

**Comment by GunHog** [comment]
That armor nerf PR should never have been merged in the first place, in my opinion. These PRs are about repairing the damage it did to the codebase.

**Comment by Anonus** [comment]
Ah yes, the eternal armor argument. This is at least a step in the right direction for ling.

**Comment by optimumtact** [comment]
ling should have a very powerful go loud, but it should take time to get the chems for it.

wasn't that the whole point of the original gamemode?

**Comment by Iamgoofball** [comment]
@optimumtact Yeah, the problem was, his "go loud" was worse than the shit you can easily get out of the armory.

**Comment by GunHog** [comment]
> ling should have a very powerful go loud, but it should take time to get the chems for it.

Well, the 'whole sucking people off for ex-pees' was removed, so you start with 10 max DNA, you end with 10 max DNA. Someone did not like lings leaving corpse piles and having a one-click, stealthy GG (not that they should), and decided that lings should poop spiders instead!

**Comment by Incomptinence** [comment]
Biggest problem wasn't that evolution points incentivized mass murder it was that the original absorb objective values and ling count together meant that collectively the lings might need an amount of dna equaling the entire station population or monkey farming. Either case absorbing is slow it slows you down doing it like if every traitor had needed to do a manditory tbagging of every kill to get green text and had to kill over ten people. People weren't killing for the powers alone absorb objective mandated it.

**Comment by Supermichael777** [comment]
arm-blade chem-sacks armor sheild adrenilen.

look at me mom I'm unstoppable

**Comment by Iamgoofball** [comment]
@Supermichael777 Still better than what we have now.

I feel like the space suit might be a bit too much. Let's wait until this is merged and feedback acquired till we do anything.

**Comment by GunHog** [comment]
<b>poke</b>

**Comment by Fox-McCloud** [comment]
just make the armor values identical, IMO, @Iamgoofball .

**Comment by GunHog** [comment]
Description needs updating, and the values should be made consistent.

**Pull request merged**